_data/sidebars/sidebar.yml

```diff
@@ -14,11 +14,7 @@ entries:
       type: homepage
       output: web
 
-    - title: Change Log
-      url: /changelog
-      output: web
-
-    - title: Introduction
+    - title: Usage example
```

Comment:
Yes, separate page would be good.
The thing what hesitated me that on the site, in Overview we have two similar pages, Get started and Introduction
It is synonym, as far as I understand, and looks strange.
A little bit confusing, but really a cosmetic issue